src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SemanticFacts/CSharpSemanticFacts.cs

```diff
@@ -417,7 +421,7 @@ SyntaxKind.DefineDirectiveTrivia or
     public bool TryGetPrimaryConstructor(INamedTypeSymbol typeSymbol, [NotNullWhen(true)] out IMethodSymbol? primaryConstructor)
         => typeSymbol.TryGetPrimaryConstructor(out primaryConstructor);
 
-#if !CODE_STYLE
+#if CSHARP_WORKSPACE
```

Comment:
RoslynAnalyzers is also "!CODE_STYLE".  so that breaks badly when sharing these files.   i didn't feel comfortable defining that constnat for RoslynAnalyzers.  So instead, i just flipped these to be either `WORKSPACE` or `!WORKSPACE` so that the right semantics are there for the workspace layer vs anyone else.